tools/sense_studio/templates/record_video.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'skeleton.html' %}
+
+{% block title %} Record video {% endblock %}
+
+{% block main %}
+<div id="main" class="ui main text container">
+    <h1 class="ui header">
+        Video List
+        <div class="sub header">
+            <i class="folder open icon"></i> {{ path }}
+            <br>
+            <i class="eye icon"></i> {{ label }}
+            <br>
+            <i class="database icon"></i> {{ split }}
+        </div>
+    </h1>
+    <div>
+        <a>Pre recording duration</a>
+        <input name="duration_selection" type="text"  id="pre_recording_duration" value="3" />
+        <p/>
+        <a>Recording duration</a>
+        <input name="duration_selection" type="text"  id="recording_duration" value="5"/>
+        <p/>
+        <button onclick="record_video()">record</button>
```

Comment:
Old style inputs and button. Please use Semantic UI elements.
![image](https://user-images.githubusercontent.com/5707898/106746775-460bd080-6623-11eb-9689-7e70fa104a8e.png)